TransferMethod/Sources/View/Widget/AbstractWidget.swift

```diff
@@ -127,9 +128,27 @@ class AbstractWidget: UIStackView, UITextFieldDelegate {
 
     func textFieldDidBeginEditing(_ textField: UITextField) {
         textField.placeholder = nil
+        if let isValueMasked = field.fieldValueMasked, isValueMasked && !isValueUpdated {
+            textField.selectedTextRange = textField.textRange(from: textField.beginningOfDocument,
+                                                              to: textField.endOfDocument)
+        }
+    }
+
+    func textField(_ textField: UITextField,
+                   shouldChangeCharactersIn range: NSRange,
+                   replacementString string: String) -> Bool {
+        if let isValueMasked = field.fieldValueMasked, isValueMasked && !isValueUpdated {
+            textField.text = ""
+            isValueUpdated = true
+        }
+        return true
     }
 
     func textFieldDidEndEditing(_ textField: UITextField) {
+        if let isValueMasked = field.fieldValueMasked, isValueMasked && !isValueUpdated {
+            return
+        }
+        isValueUpdated = true
```

Comment:
This is not required here. It should be set to true in `func textField(_ textField: UITextField,
                   shouldChangeCharactersIn range: NSRange,
                   replacementString string: String) -> Bool`  for all other fields as well